R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
                                        spark_versions = NULL,
```

Comment:
I think we need to provide a default here which documents what we know is necessary (e.g. we know that moving from spark 1.6 to 2.0 likely requires a new JAR due to the change in type of DataFrame).